GameData.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Data of application.
+    /// </summary>
+    public class GameData
+    {
+        readonly public int[,] data;
+        readonly public int[,] newData;
+
+        /// <summary>
+        /// Data of application.
+        /// </summary>
+        public GameData()
+        {
+            int column = UserInterface.GetValueInRange("Input amount of columns", 5, 200);
+            int row = UserInterface.GetValueInRange("Input amount of rows", 5, 200);
+            data = new int[row, column];
+            newData = new int[row, column];
+            FillDefault();
+        }
+
+        /// <summary>
+        /// Fills gameboard with random alive cells.
+        /// </summary>
+        void FillDefault()
+        {
+            Random random = new Random();
+            for (int i = 0; i < data.GetLength(0); i++)
+            {
+                for (int j = 0; j < data.GetLength(1); j++)
+                {
+                    data[i, j] = random.Next(0, 4) == 0 ? 1 : 0;
+                }
+            }
+        }
+        
+        /// <summary>
+        /// Iterates to the next generation.
+        /// </summary>
+        public void Iterate()
```

Comment:
Iterate what?

Reply:
Changed function name and summary to better represent its meaning.